guides/common/modules/proc_creating-a-playbook-with-modules-from-project-ansible-collection.adoc

```diff
@@ -19,6 +19,7 @@ The following examples use Ansible vault to manage sensitive data.
 * A system that can reach the {Project} API.
 This is the system that you will execute your playbook against.
 It can be the {ProjectServer} itself, a {SmartProxyServer}, or any other system in your environment.
```

Comment:
I propose the following change: https://github.com/theforeman/foreman-documentation/pull/3882/files